### ISSUE TITLE
Donor should able to accept donation after completing PayPal Donations onboarding process

### ISSUE THREAD

**Issue by ravinderk** [impress-org/givewp#5481]
## User Story

As an admin, I want to accept donations with PayPal donations after onboarding.

## Details

Few plugin users are reporting that they are unable to accept donations after successful onboarding. There possibility that we are not handling an error user care when onboarding a PayPal account and they are getting the false status of successful onboarding.

## Additional Context
On basis of few params in the PayPal return URL, we decide that the PayPal account connected or not. Maybe this is causing an error. I think we can validate the connection with PayPal API before processing the next step in client onboarding. 

## Acceptance Criteria

-   [ ] We are using the PayPal API to confirm PayPal account connection status instead of verifying it on basis of PayPal return URL query param.

**Comment by ravinderk** [comment]
@JasonTheAdams I reviewed list of error which plugin users are facing. I find out that most of issue happening because we did not cleanup merchant detail when seller status confirmed negative from PayPal instead of that we only show them error. Because of merchant detail and green icon below `Connected` button, admin gets  an impression of connected PayPal account.

![image](https://user-images.githubusercontent.com/1784821/102251126-6627e900-3f2a-11eb-86c6-78ab6eada624.png)

To solve this issue i suggest to edit UI which give an impression of connected account for unsuccessful onboarding.
